annual/CFR-2013-title37-vol1-part1.xml

```diff
@@ -1934,9 +1934,11 @@
           </GPOTABLE>
           <P>§ 1.217—for processing a redacted copy of a paper submitted in the file of an application in which a redacted copy was submitted for the patent application publication.</P>
           <P>§ 1.221—for requesting voluntary publication or republication of an application.</P>
-          <P>(j) [Reserved]</P>
           <GPOTABLE CDEF="s50,10" COLS="2" OPTS="L0,tp0,p0,8/9,g1,t1">
             <ROW>
+              <ENT>(j) [Reserved]</ENT>
+            </ROW>
+            <ROW>
```

Comment:
It might make more sense to pop (k), (l), etc. out in the long run as they're not addressable otherwise. Something to consider!